### PR TITLE
Add home path on the config

### LINKED ISSUE #426: VATZ Config - Put home_path into config file.
## Checklist

- [ ] New Feature for the Service/Plugin (Vatz)
- [x] Enhancement (Vatz)
- [ ] others(etc. e.g, documentation, project policy management)

---

Put home_path into config_file

### PR DESCRIPTION
**Pull request by rootwarp**
### 1. Type of change

Please delete options that are not relevant.

- [x] New feature
- [ ] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [ ] others (anything other than above)

---

**Related:** # (issue)

close #426 

**Summary**

Add home directory of VATZ on config file and all related commands load vatz home from config.

---

### CODE REVIEW

**Comment by rootwarp** [comment]
```
ok      github.com/dsrvlabs/vatz/utils  1.619s
?       github.com/dsrvlabs/vatz        [no test files]
?       github.com/dsrvlabs/vatz/manager/types  [no test files]
?       github.com/dsrvlabs/vatz/mocks  [no test files]
ok      github.com/dsrvlabs/vatz/cmd    3.475s  coverage: 20.2% of statements
ok      github.com/dsrvlabs/vatz/manager/api    0.553s  coverage: 0.0% of statements [no tests to run]
ok      github.com/dsrvlabs/vatz/manager/config 0.832s  coverage: 79.4% of statements
ok      github.com/dsrvlabs/vatz/manager/dispatcher     1.628s  coverage: 7.8% of statements
ok      github.com/dsrvlabs/vatz/manager/executor       1.064s  coverage: 68.9% of statements
ok      github.com/dsrvlabs/vatz/manager/healthcheck    1.904s  coverage: 46.5% of statements
ok      github.com/dsrvlabs/vatz/manager/plugin 7.286s  coverage: 51.1% of statements
ok      github.com/dsrvlabs/vatz/monitoring/prometheus  1.779s  coverage: 0.0% of statements
ok      github.com/dsrvlabs/vatz/rpc    2.509s  coverage: 67.2% of statements
ok      github.com/dsrvlabs/vatz/utils  1.511s  coverage: 7.1% of statements
```

File: cmd/init.go [hunk 1]
```diff
@@ -41,6 +43,7 @@ func createInitCommand(initializer tp.Initializer) *cobra.Command {
     address: "127.0.0.1"
     grpc_port: 19090
     http_port: 19091
+  home_path: "%s"
```
Comment (xellos00) [inline cmd/init.go:46]: @rootwarp 
how about vatz_home_path? 
what do you think?

Comment (rootwarp) [reply cmd/init.go:46]: @xellos00 That's better, I'll stack the new commit soon. 👍

**Review by heejin-github — APPROVED** [review]
LGTM

File: cmd/init.go (hunk 1 above)
Comment (xellos00) [reply cmd/init.go:46]: > @xellos00 That's better, I'll stack the new commit soon. 👍

@rootwarp 
any update?

Comment (rootwarp) [reply cmd/init.go]: @xellos00 Oh; Sorry, I missed this.
I push the new commit. can you please check this again?

Comment (xellos00) [reply cmd/init.go]: > @xellos00 Oh; Sorry, I missed this. I push the new commit. can you please check this again?

Great!!

**Comment by xellos00** [comment]
@gnongs 
Can you verify this review?

**Review by gnongs — APPROVED** [review]
LGTM!

**Pull request merged**